spring-integration-syslog/src/main/java/org/springframework/integration/syslog/RFC5424SyslogParser.java

```diff
@@ -152,6 +155,7 @@ protected Object getTimestamp(Reader r) {
 		return dateBuilder.toString();
 	}
 
+	@Nullable
 	private Object getStructuredData(Reader r) {
```

Comment:
Please, put `@Nullable` on the return type.
This is a contract exactly of the return type, not a method.